src/hashstore/hashstoreclient.py

```diff
@@ -234,11 +245,11 @@ class HashStoreClient:
     MET_TYPE = "metadata"
 
     def __init__(self, properties, testflag=None):
-        """Initialize HashStore and MetacatDB
+        """Store objects in a given directory into HashStore.
 
-        Args:
-            properties: See FileHashStore for dictionary example
-            testflag (str): "knbvm" to initialize MetacatDB
+        :param str origin_dir: Directory to convert.
+        :param str obj_type: Type of objects ('object' or 'metadata').
+        :param int num: Number of files to store.
```

Comment:
none of these params match the method signature

Reply:
D'oh! I have revised the doc string.